modules/services/images/src/main/res/drawable/shuffle.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="25dp"
+    android:height="25dp"
```

Comment:
It's a bit strange that the SVG has come out of Figma at 25dp x 25dp, as they are normally 24dp. It's not a big deal, but it might be worth asking the designers.

Reply:
Does it matter for vectors? We can always use whichever size we want in the Kotlin/XML code.

Reply:
As most of the SVGs in the project are 24dp and in Figma it says 24dp I would ask why it's coming out as 25dp. There could be an issue the designers don't know about. 

<img width="254" alt="Screenshot 2024-10-22 at 4 28 35 pm" src="https://github.com/user-attachments/assets/5f937181-491e-4305-bba7-2514a8fd3c1d">

Reply:
I think this started recently. I don’t remember seeing this issue as often before